2020_Fall/readme.md

```diff
@@ -59,7 +59,7 @@ Make the Pull Request from your own GitHub account. Make sure you're changing th
 
 - [ ] Huang, Valerie
 
-- [ ] Jenkins, Michael
+- [x ] Jenkins, Michael 🇬🇧
```

Comment:
I think you've got an extra space there after the `x`. Just make another edit (which will create another commit on this branch).